src/libzip-windows/libzip-windows.projitems

```diff
@@ -1,15 +1,28 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition=" '$(HostOS)' == 'Linux' ">
+    <CMakeBinary32>\usr\bin\cmake</CMakeBinary32>
```

Comment:
Why explicitly qualify `cmake` instead of relying on `$PATH`, as is done for macOS?

Reply:
We also build cmake as part of mxe and I want to be explicit that we run (we MUST run) the system one because it's the "native" one and we're configuring a cross build using a "native" toolchain (mingw). Linux currently doesn't build mxe (Ubuntu and friends) but it's not unlikely that some distro will need to build it for one reason or another. I don't want to have any ambiguities